core.cc

```diff
@@ -30,7 +30,7 @@ class Worker: public Nan::AsyncWorker {
         void HandleOKCallback() {
             Nan::HandleScope scope;
 
-            if(!callback->IsEmpty())
+            if(callback->GetFunction()->IsFunction())
                 callback->Call(0, NULL);
         };
```

Comment:
Not following. Check my repush. callback argument is mandatory to Worker, and the default implement of of HandleOKCallback just blindly uses it.

When I check IsFunction(), I have no callback, so don't want to cast the non-existent argument to a function, or in any way create a callback, but I have to pass something in as cb argument to the constructor.